### DISCUSSION TITLE
Label issues and use discussions for questions

### DISCUSSION THREAD

**Discussion by Abdeldjalil-H** [tortoise/tortoise-orm#1772] [Ideas]
First of all, thank you for this awesome ORM. I have some suggestions to make it easier for contributors.

**Add labels to issues:** There are already many labels for issues, however most issues are not labeled except for some older ones. I believe that adding labels helps prioritize issues and makes it easier for contributors to find issues to work on.

**Convert questions into discussions:** I have noticed that many issues in this project are actually questions, possibly because discussions weren't supported at that time. I think it would be beneficial to convert these into discussions, similar to what was done with FastAPI. This will help contributors focus on actual issues and also allow marking correct answers. The process could start by:
1. Adding the `question` label to these posts
2. Bulk converting them into discussions
3. Adding a checklist to the issue template to inform newcomers that they should use discussions for questions.

@abondar @henadzit

**Comment by henadzit** [comment]
@Abdeldjalil-H these are all great points, this is definitely something we are planing to do!
